schema/deploy/computed_columns/application_revision_total_ciip_emissions.sql

```diff
@@ -0,0 +1,24 @@
+-- Deploy ggircs-portal:computed_columns/application_revision_facility_emission to pg
+-- requires: tables/gas_001
+-- requires: views/ciip_emission
+
+begin;
+
+-- Computed column returns the sum of (ciip) emissions for an application revision.
+create or replace function ggircs_portal.application_revision_total_ciip_emissions(application_revision ggircs_portal.application_revision)
+returns numeric
+  as $$
+    select sum(annual_co2e) as total_ciip_co2e_emissions from ggircs_portal.ciip_emission ce
+    join ggircs_portal.gas
+    on ce.gas_type = gas.gas_type
+    and ce.version_number = application_revision.version_number
+    and ce.application_id = application_revision.application_id
+    and ce.gas_type != 'CO2'
```

Comment:
The CO2 gas type never actually made it to prod, so I don't think that we need to filter it out.
```suggestion
```